dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -615,6 +618,22 @@ private void createXMLFiles() throws ServletException, IOException, DotDataExcep
 				int total =0;
 				java.text.NumberFormat formatter = new java.text.DecimalFormat("0000000000");
 				/* we will only export 10,000,000 items of any given type */
+				
+
+				for(i=0;i < 10000000;i=i+step){
+					List<Tree> trees = TreeFactory.getAllTrees(step * i, step);
+					if(trees==null || trees.isEmpty()){
+						break;
+					}
+					File f  = new File(backupTempFilePath + "/" + Tree.class.getName() + "_" + formatter.format(i) + ".xml");

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)